README.md

```diff
@@ -52,7 +52,6 @@ You can run this repo virtually by using GitHub Codespaces or VS Code Remote Con
 1. Create a new folder and switch to it in the terminal
 1. Run `azd auth login`
 1. Run `azd init -t azure-search-openai-demo`
-    * For the target location, the regions that currently support the models used in this sample are **East US** or **South Central US**. For an up-to-date list of regions and models, check [here](https://learn.microsoft.com/en-us/azure/cognitive-services/openai/concepts/models)
```

Comment:
Can you move the location note down? It's really easy to have a bad deploy if you've picked a bad location, so it should be noted in 'azd up' section.

Reply:
done!